deploy.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -euo pipefail
+set -eo pipefail
```

Comment:
Why remove `u`? looks like we don't expect any unbound variables in this script

Reply:
It is an issue with virtualenv which is failing this script on `./venv/bin/activate: line 57: PS1: unbound variable`

Reply:
ugh, that's nasty.
in the case, I'd surround that line with
`set +u`
and
`set -u`

Reply:
👍 Updated to have it only around `./venv/bin/activate`